receiver/azureeventhubreceiver/README.md

```diff
@@ -24,18 +27,67 @@ The offset at which to start watching the event hub. If empty, it starts with th
 
 Default: ""
 
-Example:
+### format (Optional)
+Determines how to transform the Event Hub messages into OpenTelemetry logs. See the "Format"
+section below for details.
+
+Default: "raw"
```

Comment:
I believe we discussed making "azure" the default, eventually. Do we have an issue to track this?